app/src/androidTest/java/com/epam/keystore/ThemisReadWriteInstrumentedTest.java

```diff
@@ -0,0 +1,107 @@
+package com.epam.keystore;
+
+import android.content.Context;
+import android.support.test.InstrumentationRegistry;
+import android.support.test.runner.AndroidJUnit4;
+
+import com.epam.keystore.core.SecurityProvider;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotEquals;
+import static org.junit.Assert.assertNull;
+
+/**
+ * Instrumented test, which will execute on an Android device.
+ * This test need launch in two devices for 18 - 22 and 23-27 version API
+ *
+ * @see <a href="http://d.android.com/tools/testing">Testing documentation</a>
+ */
+@RunWith(AndroidJUnit4.class)
+public class ThemisReadWriteInstrumentedTest {
+    private SecureStorage storage;
+
+    @Before
+    public void before() {
+        Context context = InstrumentationRegistry.getTargetContext();
+        storage = new SecureStorage(context, SecurityProvider.Type.THEMIS);
+    }
+
+    @Test
+    public void shouldGetNullValueIfNotSet() {
+        String value = storage.get("blabla");
+        assertNull(value);
+    }
+
+    @Test
+    public void shouldSaveValue() {
+        storage.save("key", "passWORD");
+        assertEquals("passWORD", storage.get("key"));
+    }
+
+    @Test
+    public void shouldSaveOtherKeyValue() {
+        storage.save("key1", "passWORD");
+        assertEquals("passWORD", storage.get("key1"));
```

Comment:
what will happens if two values are set to the same key?

like this

```
storage.save("key1", "value1");
storage.save("key1", "value2");
storage.get("key1")
```

should we add test for this?